org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/Types.java

```diff
@@ -193,19 +193,19 @@ public FunctionBreakpoint(String name) {
     public static class ExceptionBreakpointFilter {
         public static final String Filter_All = "all";
         public static final String Filter_UserUnhandled = "user-unhandled";
+        public static final String Filter_UserHandled = "user-handled";
 
         public String label;
         public String filter;
-
-        /**
-         * Constructor.
-         */
-        public void setFilter(String value) {
-            if (!Filter_All.equals(value) && !Filter_UserUnhandled.equals(value)) {
-                return;
-            }
+        
+        public ExceptionBreakpointFilter(String value, String label) {
             this.filter = value;
+            this.label = label;
         }
+        
+        public static ExceptionBreakpointFilter All = new ExceptionBreakpointFilter(Filter_All, "All Exceptions");
```

Comment:
Please see the following:
https://github.com/Microsoft/vscode-debugadapter-node/blob/master/protocol/src/debugProtocol.ts#L1351
export type ExceptionBreakMode = 'never' | 'always' | 'unhandled' | 'userUnhandled';

Please help me understand the difference between the filters and the break mode definition used in ExceptionOptions. If I want to break whenever there's an unhandled exception, what is the workflow?